+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri May 26 06:51:57 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL 'atune atune-engine'
+ pkgs='atune atune-engine'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs 'atune atune-engine' --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpo7_g7frm
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpo7_g7frm
++ lsblk
++ grep disk
++ awk 'NR==1{print $1}'
+ disk_name=vda
+ sed -i 's\disk = sda\disk = vda\g' /etc/atuned/atuned.cnf
+ sed -i 's\network = enp189s0f0\network = \g' /etc/atuned/atuned.cnf
+ sed -i 's\rest_tls = true\rest_tls = false\g' /etc/atuned/atuned.cnf
+ sed -i 's\engine_tls = true\engine_tls = false\g' /etc/atuned/atuned.cnf
+ sed -i 's\engine_tls = true\engine_tls = false\g' /etc/atuned/engine.cnf
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri May 26 06:52:37 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri May 26 06:52:37 2023 - INFO  - Start testing...
+ test_execution atuned.service
+ service=atuned.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-05-26 06:52:37'
+ test_restart atuned.service
+ service=atuned.service
+ systemctl restart atuned.service
Failed to restart atuned.service: Unit polkit.service not found.
+ CHECK_RESULT 5 0 0 'atuned.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='atuned.service restart failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'atuned.service restart failed'
+ LOG_ERROR 'atuned.service restart failed'
+ message='atuned.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'atuned.service restart failed'
Fri May 26 06:52:37 2023 - ERROR - atuned.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Fri May 26 06:52:37 2023 - ERROR - ../common/common_lib.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status atuned.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'atuned.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='atuned.service restart failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'atuned.service restart failed'
+ LOG_ERROR 'atuned.service restart failed'
+ message='atuned.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'atuned.service restart failed'
Fri May 26 06:52:42 2023 - ERROR - atuned.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Fri May 26 06:52:42 2023 - ERROR - ../common/common_lib.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop atuned.service
+ CHECK_RESULT 0 0 0 'atuned.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='atuned.service stop failed'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status atuned.service
+ grep 'Active: inactive'
     Active: inactive (dead)
+ CHECK_RESULT 0 0 0 'atuned.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='atuned.service stop failed'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl start atuned.service
Failed to start atuned.service: Unit polkit.service not found.
+ CHECK_RESULT 5 0 0 'atuned.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='atuned.service start failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'atuned.service start failed'
+ LOG_ERROR 'atuned.service start failed'
+ message='atuned.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'atuned.service start failed'
Fri May 26 06:52:47 2023 - ERROR - atuned.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Fri May 26 06:52:47 2023 - ERROR - ../common/common_lib.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status atuned.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'atuned.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='atuned.service start failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'atuned.service start failed'
+ LOG_ERROR 'atuned.service start failed'
+ message='atuned.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'atuned.service start failed'
Fri May 26 06:52:52 2023 - ERROR - atuned.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Fri May 26 06:52:52 2023 - ERROR - ../common/common_lib.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ test_enabled atuned.service
+ service=atuned.service
++ systemctl is-enabled atuned.service
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable atuned.service
++ grep 'Created symlink'
++ awk '{print $3}'
+ symlink_file=/etc/systemd/system/multi-user.target.wants/atuned.service
+ find /etc/systemd/system/multi-user.target.wants/atuned.service
/etc/systemd/system/multi-user.target.wants/atuned.service
+ CHECK_RESULT 0 0 0 'atuned.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='atuned.service enable failed'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable atuned.service
Removed /etc/systemd/system/multi-user.target.wants/atuned.service.
+ find /etc/systemd/system/multi-user.target.wants/atuned.service
find: ‘/etc/systemd/system/multi-user.target.wants/atuned.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'atuned.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='atuned.service disable failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ journalctl --since '2023-05-26 06:52:37' -u atuned.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of atuned.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of atuned.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload atuned.service
+ service=atuned.service
+ systemctl start atuned.service
Failed to start atuned.service: Unit polkit.service not found.
+ systemctl reload atuned.service
+ grep 'Job type reload is not applicable'
Failed to reload atuned.service: Job type reload is not applicable for unit atuned.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit atuned.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit atuned.service'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status atuned.service
+ grep 'Active: active'
+ systemctl status atuned.service
+ grep 'inactive (dead)'
     Active: inactive (dead)
+ systemctl status atuned.service
+ grep 'Condition check'
+ grep skip
+ CHECK_RESULT 1 0 0 'atuned.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='atuned.service reload causes the service status to change'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'atuned.service reload causes the service status to change'
+ LOG_ERROR 'atuned.service reload causes the service status to change'
+ message='atuned.service reload causes the service status to change'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'atuned.service reload causes the service status to change'
Fri May 26 06:52:53 2023 - ERROR - atuned.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 95'
+ message='../common/common_lib.sh line 95'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 95'
Fri May 26 06:52:53 2023 - ERROR - ../common/common_lib.sh line 95
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri May 26 06:52:53 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 5
+ test 5 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri May 26 06:52:53 2023 - ERROR - failed to execute the case.
+ exit 5
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri May 26 06:52:53 2023 - INFO  - start environment cleanup.
+ systemctl stop atuned.service
+ sed -i 's\disk = vda\disk = sda\g' /etc/atuned/atuned.cnf
+ sed -i 's\network = \network = enp189s0f0\g' /etc/atuned/atuned.cnf
+ sed -i 's\rest_tls = false\rest_tls = true\g' /etc/atuned/atuned.cnf
+ sed -i 's\engine_tls = false\engine_tls = true\g' /etc/atuned/atuned.cnf
+ sed -i 's\engine_tls = false\engine_tls = true\g' /etc/atuned/engine.cnf
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpo7_g7frm ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpo7_g7frm
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri May 26 06:52:55 2023 - INFO  - Finish environment cleanup!
